charts/seed-monitoring/charts/grafana/dashboards/owners/vpn-dashboard.json

```diff
@@ -518,24 +518,24 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "rate(container_cpu_usage_seconds_total{pod=~\"kube-apiserver-.*|prometheus-.*\", container=\"vpn-seed\"}[$__rate_interval])",
+          "expr": "rate(container_cpu_usage_seconds_total{pod=~\"vpn-seed-.*\", container=\"vpn-seed-server\"}[$__rate_interval])",
           "format": "time_series",
           "hide": false,
-          "interval": "10s",
+          "interval": "",
           "intervalFactor": 1,
           "legendFormat": "Actual ({{pod}}/{{container}})",
           "refId": "A",
           "step": 20
         },
         {
-          "expr": "kube_pod_container_resource_requests{resource=\"cpu\", unit=\"core\", pod=~\"kube-apiserver-.*|prometheus-.*\", container=\"vpn-seed\"}",
+          "expr": "kube_pod_container_resource_requests{resource=\"cpu\", unit=\"core\", pod=~\"vpn-seed-.*\", container=\"vpn-seed-server\"}",
```

Comment:
Does this also work on HA VPN case when there are 2 VPN pods?

Reply:
As far as I can tell, it will show the values of the `vpn-seed-server` pods in HA VPN case as well. However, it will not add the `vpn-client` containers of `kube-apiserver` in the HA VPN case. It is probably a good idea to add this as well.

/cc @MartinWeindel

Reply:
Yes, it will show all `vpn-seed-server` pods
> Does this also work on HA VPN case when there are 2 VPN pods?

Reply:
@Sallyan Could you please also add the vpn-client containers of `kube-apiserver` in the HA VPN scenario?

Reply:
@ScheererJ  we could see the `vpn-client` containers of `kube-apiserver` from another dashboard `Shoot control plane resource usage by owner and container`. Do you think it is more meaningful to add the `vpn-client` resource usage in dashboard VPN?
<img width="237" alt="Screenshot 2023-05-16 at 17 54 37" src="https://github.com/gardener/gardener/assets/50126000/a9f7f4b1-fc61-4f01-a6e5-2c02f844d470">

<img width="1414" alt="Screenshot 2023-05-16 at 17 55 05" src="https://github.com/gardener/gardener/assets/50126000/8e262b86-68cd-4466-ba8b-9b3611a7d415">